Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs

```diff
@@ -106,6 +107,37 @@ protected override void AddChildView(VisualElement childView)
 			}
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			if (_disposed)
+			{
+				return;
+			}
+
+			_disposed = true;
```

Comment:
It's possible (nay, likely) that there are places in the code where we've done this incorrectly; a comprehensive review of our `Dispose()` implementations is on my short list.